be/src/olap/memtable.cpp

```diff
@@ -406,6 +407,22 @@ Status MemTable::flush() {
     DorisMetrics::instance()->memtable_flush_duration_us->increment(duration_ns / 1000);
```

Comment:
I suggest that put code generating delete bitmap in a function, e.g. generate_delete_bitmap, and  metric memtable_flush_duration_us should include time consumed to delete bitmap.

Reply:
done